dashboard/client/src/common/ServeStatus.tsx

```diff
@@ -0,0 +1,77 @@
+import { createStyles, makeStyles } from "@material-ui/core";
+import classNames from "classnames";
+import React from "react";
+import {
+  RiCheckboxCircleFill,
+  RiCloseCircleFill,
+  RiStopCircleFill,
+} from "react-icons/ri";
+import { ServeApplication } from "../type/serve";
+import { JobRunningIcon } from "./JobStatus";
+import { ClassNameProps } from "./props";
+
+type ServeStatusIconProps = {
+  app: ServeApplication;
+  small: boolean;
+} & ClassNameProps;
+
+const useServeStatusIconStyles = makeStyles((theme) =>
+  createStyles({
+    icon: {
+      width: 20,
+      height: 20,
+      marginRight: 8,
+    },
+    iconSmall: {
+      width: 16,
+      height: 16,
+    },
+    colorSuccess: {
+      color: theme.palette.success.main,
+    },
+    colorError: {
+      color: theme.palette.error.main,
+    },
+  }),
+);
+
+export const ServeStatusIcon = ({
+  app,
+  small,
+  className,
+}: ServeStatusIconProps) => {
+  const classes = useServeStatusIconStyles();
+
+  switch (app.status) {
+    case "RUNNING":
+      return (
+        <RiCheckboxCircleFill
+          data-testid="serve-status-icon"
+          className={classNames(classes.icon, classes.colorSuccess)}
+        />
```

Comment:
@scottsun94 , do we want a green checkbox for running? That's a bit weird to me. Makes me feel like the serve application is "completed".

Reply:
Did we make a decision here?

Reply:
This is a good point. Was think about this before forgot to dig in.
Browsed the icon library and here are some options. Which option do you prefer?

option 1
<img width="574" alt="Screen Shot 2023-05-05 at 12 18 57 PM" src="https://user-images.githubusercontent.com/9677264/236550581-81f93f42-930f-463b-b14f-da41645c522e.png">

option 2
<img width="557" alt="Screen Shot 2023-05-05 at 12 19 02 PM" src="https://user-images.githubusercontent.com/9677264/236550597-09e02284-31e3-4ca8-b921-bc3bb1f1c5d8.png">

option 3
<img width="502" alt="Screen Shot 2023-05-05 at 12 18 46 PM" src="https://user-images.githubusercontent.com/9677264/236550677-e92b2eee-ca68-402d-866b-69f5340ea723.png">

option 4
<img width="565" alt="Screen Shot 2023-05-05 at 12 18 36 PM" src="https://user-images.githubusercontent.com/9677264/236550711-f6430fbc-5751-42c0-b520-3b3b23218cfc.png">

option 5
<img width="579" alt="Screen Shot 2023-05-05 at 12 18 52 PM" src="https://user-images.githubusercontent.com/9677264/236550540-ad08f814-c1af-4643-b4de-29b4ffb0f0e7.png">

Reply:
I like option 2

Reply:
RiRecordCircleFill

Reply:
Option 2 LGTM.

Reply:
<img width="634" alt="image" src="https://user-images.githubusercontent.com/125417081/236578705-95220711-0af7-4f8a-9200-ff2328b32af8.png">